plugins/System/DeviceChangeNotifier.py

```diff
@@ -1236,16 +1229,12 @@ def __call__(self, pattern=None, **kwargs):
         configuration panel, you just need to click on the device type you
         want to know about and the help will load.
 
-        :param pattern: str() text to search for.
-
         GetDevices('SOME HDD NAME')
-
-        :param kwargs: Keyword must be a valid device type. Value is the string
-         to search for.
-
         GetDevices(DiskDrive='SOME HDD NAME')
 
-        :return: tuple() WMI instances that represent a device.
+        :type pattern: str
+        :type kwargs: dict
+        :rtype: tuple
```

Comment:
Hmm, seems like we misunderstood us here. I didn't meant to remove the description of the parameter, just change the way the type is declared. I'm sorry for my vague description.

Instead of writing `str()` before the description:
```rest
:param pattern: str() text to search for.
:param kwargs: Keyword must be a valid device type. Value is the string to search for.
:return: tuple() WMI instances that represent a device.
```
add a line with the type for the parameter:
```rest
:param pattern: text to search for.
:type pattern: str
:param kwargs: Keyword must be a valid device type. Value is the string to search for.
:type kwargs: str
:return: WMI instances that represent a device.
:rtype: tuple of WMI
```
or alternative put the type before the parameter name:
```rest
:param str pattern: text to search for.
:param str kwargs: Keyword must be a valid device type. Value is the string to search for.
:return: WMI instances that represent a device.
:rtype: tuple of WMI
```
---
**To show the difference** (i've removed the long description here to make the pics smaller):

your first version:
![grafik](https://cloud.githubusercontent.com/assets/6463113/24071227/414cb096-0bcd-11e7-827f-113cc5fd1d6c.png)

with my changes:
![grafik](https://cloud.githubusercontent.com/assets/6463113/24071220/24215dfa-0bcd-11e7-85f2-49a83dc9189d.png)

what you have now:
![grafik](https://cloud.githubusercontent.com/assets/6463113/24071261/1256e9e0-0bce-11e7-8fe1-36fac70b4733.png)